core/cloudflow-crd/src/main/scala/akka/datap/crd/App.scala

```diff
@@ -50,6 +52,15 @@ object App {
   final val ProtocolVersionKey = "protocol-version"
   final val ProtocolVersion = "7"
 
+  val customResourceDefinitionVersion: CustomResourceDefinitionVersion = {
+    new CustomResourceDefinitionVersionBuilder()
+      .withName(GroupVersion)
+      .withNewSubresources()
+      .withNewStatus()
+      .endStatus()
```

Comment:
Hi @thomasschoeftner. Many thanks for the PR. Quick question, why is this `endStatus` needed? or the next `endSubresources`

Reply:
I see now what it is. No worries.